packages/salesforcedx-vscode-lwc/package.nls.ja.json

```diff
@@ -6,6 +6,7 @@
   "lightning_lwc_preferences": "Salesforce Lightning Web Components",
   "lightning_lwc_preview_text": "SFDX: コンポーネントをローカルでプレビュー",
   "lightning_lwc_remember_device_description": "最後に使用したモバイルデバイスのターゲットを保存する",
+  "lightning_lwc_preview_typescript_support": "最後に使用したモバイルデバイスのターゲットを保存する",
```

Comment:
I will contact the IDEx doc writer to get an official description, this is a placeholder for now.

Reply:
Spoke with doc writers and this change is not needed.